migrations/175_workspace_logs__create.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TYPE enum_workspace_log_level AS ENUM ('error', 'warn', 'info', 'debug');
```

Comment:
I guess I was thinking to make this just `log_level`, so that it's usable by other log tables in the future. I think these levels are pretty generic.